AspNetCore.sln

```diff
@@ -7205,7 +7213,6 @@ Global
 		{010A9638-F20E-4FE6-A186-85732BFC9CB0}.Release|x64.Build.0 = Release|Any CPU
 		{010A9638-F20E-4FE6-A186-85732BFC9CB0}.Release|x86.ActiveCfg = Release|Any CPU
 		{010A9638-F20E-4FE6-A186-85732BFC9CB0}.Release|x86.Build.0 = Release|Any CPU
-=======
```

Comment:
I was particularly surprised that this didn't cause issues with the CI or for anyone else working with the various slnf files.

```
aspnet\AspNetCore\src\Http [halter73/25342]> dotnet build .\HttpAbstractions.slnf
Microsoft (R) Build Engine version 16.8.0-preview-20379-02+8343d638e for .NET
Copyright (C) Microsoft Corporation. All rights reserved.

F:\dev\aspnet\AspNetCore\AspNetCore.sln(7208): Solution file error MSB5007: Error parsing the project configuration section in solution file. The entry "=======" is invalid.

Build FAILED.

F:\dev\aspnet\AspNetCore\AspNetCore.sln(7208): Solution file error MSB5007: Error parsing the project configuration section in solution file. The entry "=======" is invalid.
    0 Warning(s)
    1 Error(s)
```

I guess most people don't try this because there are sadly other issues with using `dotnet build` and `dotnet test` on slnf files in the repo right now (namely CS7035, CS7034 and CS0579), but I still think we should fix it when AspNetCore.sln is not formatted correctly.

Reply:
Sadly we don't use AspNetCore.sln on CI (yet).